contracts/smart-account/modules/EcdsaOwnershipRegistryModule.sol

```diff
@@ -106,6 +107,43 @@ contract EcdsaOwnershipRegistryModule is
         bytes memory moduleSignature,
         address smartAccount
     ) public view virtual override returns (bytes4) {
+        if (
+            _verifySignature(
+                keccak256(
+                    abi.encodePacked(
+                        "\x19Ethereum Signed Message:\n52",
```

Comment:
I think `\x19Ethereum Signed Message:\n52` should be `\x19Ethereum Signed Message:\n32` since the hash lenght is 32

Reply:
Nope, it should be 52 here, 32 bytes for hash + 20 bytes for the address

Reply:
I missed the presence of the smartAccount